Makefile

```diff
@@ -1,15 +1,20 @@
-XDR_BASE_URL_CURR=https://github.com/stellar/stellar-xdr/raw/curr
+XDR_BASE_URL_CURR=https://github.com/stellar/stellar-xdr/raw/9ac02641139e6717924fdad716f6e958d0168491
 XDR_BASE_LOCAL_CURR=xdr/curr
 XDR_FILES_CURR= \
 	Stellar-SCP.x \
 	Stellar-ledger-entries.x \
 	Stellar-ledger.x \
 	Stellar-overlay.x \
 	Stellar-transaction.x \
-	Stellar-types.x
+	Stellar-types.x \
+	Stellar-contract.x \
+	Stellar-contract-env-meta.x \
+	Stellar-contract-meta.x \
+	Stellar-contract-spec.x \
+	Stellar-contract-config-setting.x
 XDR_FILES_LOCAL_CURR=$(addprefix xdr/curr/,$(XDR_FILES_CURR))
 
-XDR_BASE_URL_NEXT=https://github.com/stellar/stellar-xdr/raw/e372df9f677961aac04c5a4cc80a3667f310b29f
+XDR_BASE_URL_NEXT=https://github.com/stellar/stellar-xdr/raw/440dc9512b6e72cf84965641c5eb495d6043ed73
```

Comment:
the convention is to generate next and curr even though respectively one is never referenced, just curious. does the dormant un-used one still get packaged into dist?

Reply:
ooh that's a great question. I think it does not get packaged because it never gets used nor exported by `src/xdr.js`, but I will double-check this!